eureka/src/main/java/com/linecorp/armeria/client/eureka/EurekaEndpointGroupBuilder.java

```diff
@@ -232,6 +237,14 @@ public EurekaEndpointGroupBuilder registryFetchIntervalMillis(long registryFetch
         return this;
     }
 
+    /**
+     * Sets whether to store {@link InstanceInfo#getMetadata()} as {@link Endpoint#attr(AttributeKey)}.
+     */
+    public EurekaEndpointGroupBuilder instanceMetadataAsAttrs(boolean instanceMetadataAsAttrs) {
+        this.instanceMetadataAsAttrs = instanceMetadataAsAttrs;
+        return this;
+    }
```

Comment:
What's your concern for making this configurable? I thought we could just always enable this without any opt-opt.